src/tests/ftest/container/query_attribute.py

```diff
@@ -147,6 +150,91 @@ def test_container_query_attr(self):
         self.log.debug(str(actual_attrs))
         self.assertEqual(actual_attrs, expected_attrs)
 
+    def test_container_query_attrs(self):
+        """JIRA ID: DAOS-4640
```

Comment:
JIRA ID copy-paste?

Reply:
I figured I'd leave it as that was the original ticket for adding this test. I'm just adding a new testcase...

Reply:
No worries. I don't think we have clear guidance on when to use or not use a JIRA ID like this anyway. Many tests don't have it at all. But either way, the git history is a better method to determine when cases were added/modified.